include/transformations/renaming/DistributionGenerator.h

```diff
@@ -0,0 +1,13 @@
+#ifndef INCLUDE_TRANSFORMATIONS_RENAMING_DISTRIBUTIONGENERATOR_H_
+#define INCLUDE_TRANSFORMATIONS_RENAMING_DISTRIBUTIONGENERATOR_H_
+
+#include <random>
+
+using std::discrete_distribution;
+
+static class DistributionGenerator {
```

Comment:
I think creating a separate class for just a single action is not a good idea. Since we are not going to extend `DistributionGenerator` class with any other functionality, I would suggest isolating its functionality in a function `createUniformIntGenerator` and implement it in a file like `Utils.cpp`. but it's better to discuss it in slack